applications/matter_bridge/doc/adding_bridged_matter_device.rst

```diff
@@ -0,0 +1,402 @@
+               DECLARE_DYNAMIC_CLUSTER(Clusters::BridgedDeviceBasicInformation::Id, bridgedDeviceBasicAttrs, ZAP_CLUSTER_MASK(SERVER), nullptr, nullptr),
+               DECLARE_DYNAMIC_CLUSTER(Clusters::Identify::Id, identifyAttrs, ZAP_CLUSTER_MASK(SERVER), sIdentifyIncomingCommands,
+                           nullptr) DECLARE_DYNAMIC_CLUSTER_LIST_END;
+
+            DECLARE_DYNAMIC_ENDPOINT(bridgedPressureEndpoint, bridgedPressureClusters);
+
+            static constexpr uint8_t kBridgedPressureEndpointVersion = 2;
+
+            static constexpr EmberAfDeviceType kBridgedPressureDeviceTypes[] = {
+               { static_cast<chip::DeviceTypeId>(PressureSensorDevice::kPressureSensorDeviceTypeId),
+               kBridgedPressureEndpointVersion },
+               { static_cast<chip::DeviceTypeId>(MatterBridgedDevice::DeviceType::BridgedNode),
+               MatterBridgedDevice::kDefaultDynamicEndpointVersion }
+            };
+
+            static constexpr uint8_t kPressureDataVersionSize = ArraySize(bridgedPressureClusters);
+
+      - Modify the constructor:
+
+         .. code-block:: C++
+
+            PressureSensorDevice::PressureSensorDevice(const char *nodeLabel) : MatterBridgedDevice(nodeLabel)
+            {
+                  mDataVersionSize = kPressureDataVersionSize;
+                  mEp = &bridgedPressureEndpoint;
+                  mDeviceTypeList = kBridgedPressureDeviceTypes;
+                  mDeviceTypeListSize = ARRAY_SIZE(kBridgedPressureDeviceTypes);
+                  mDataVersion = static_cast<DataVersion *>(chip::Platform::MemoryAlloc(sizeof(DataVersion) * mDataVersionSize));
+            }
+
+   #. Open the :file:`nrf/samples/matter/common/bridge/matter_bridged_device.h` header file again to see which methods of the :c:struct:`MatterBridgedDevice` class are purely virtual (assigned with ``=0``) and have to be overridden by the :c:struct:`PressureSensorDevice` class.
+   #. Edit the :c:struct:`PressureSensorDevice` class, in the :file:`pressure_sensor.h` header file to declare the required methods as follows:
+
+      .. code-block:: C++
+
+        uint16_t GetDeviceType() const override;
+
+        CHIP_ERROR HandleRead(chip::ClusterId clusterId, chip::AttributeId attributeId, uint8_t *buffer,
+                    uint16_t maxReadLength) override;
+        CHIP_ERROR HandleWrite(chip::ClusterId clusterId, chip::AttributeId attributeId, uint8_t *buffer) override;
+        CHIP_ERROR HandleAttributeChange(chip::ClusterId clusterId, chip::AttributeId attributeId, void *data,
+                        size_t dataSize) override;
+
+   #. Implement the body of the :c:func:`GetDeviceType` method so that it can return the device type ID for the Pressure Sensor device type, which is equal to ``0x0305``.
+      To check the device type ID for specific type of device, see Matter Device Library Specification.
```

Comment:
Did we have a link for this?

Reply:
I don't think so, as we mention it in the `matter/overview/data_model.rst` without a link as well.